docs/cvl/invariants.md

```diff
@@ -66,9 +66,12 @@ possible values of the parameters.
 
 While verifying an invariant, the Prover checks two things.  First, it checks
 that the invariant is established after calling any constructor.  Second, it checks
-that the invariant holds after the execution of any contract method, assuming
+that the invariant holds after the execution of any methods, assuming
```

Comment:
```suggestion
that the invariant holds after the execution of any method, assuming
```
Only a single method will be executed before the invariant

Reply:
Do you think we should clarify that we only refer to `public` (or `external`) methods?

Reply:
Right - I added and clarified it.